editor/assets/effects/builtin-terrain.effect

```diff
@@ -265,7 +262,7 @@ CCProgram terrain-fs %{
     s.emissive = vec3(0.0, 0.0, 0.0);
 
     #if CC_USE_LIGHTMAP && !USE_BATCHING && !CC_FORWARD_ADD
-      vec4 lightColor = texture(lightMap, luv.xy);
+      vec4 lightColor = texture(cc_lightingMap, luv.xy);
```

Comment:
Are there compatibility issues when user upgrade from old version

Reply:
> Are there compatibility issues when user upgrade from old version

it's compatibility for old version